docker-compose-prod.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+
+services:
+  argos_file_manager_prod:
+    build:
+      context: .
+      dockerfile: Dockerfile.prod
+    image: argos-file-manager-prod
+    container_name: argos_file_manager_prod
+    ports:
+      - "8080:8080"
```

Comment:
Remember to update theses ports as well.

Reply:
problem clarified.